src/commands/General/Chat Bot Info/help.ts

```diff
@@ -91,12 +103,12 @@ export default class extends SkyraCommand {
 
 	private async _fetchCommands(message: KlasaMessage) {
 		const run = this.client.inhibitors.run.bind(this.client.inhibitors, message);
-		const commands = new Map();
+		const commands = new Collection<string, SkyraCommand[]>();
 		await Promise.all(this.client.commands.map(command => run(command, true)
 			.then(() => {
 				const category = commands.get(command.category);
-				if (category) category.push(command);
-				else commands.set(command.category, [command]);
+				if (category) category.push(command as SkyraCommand);
+				else commands.set(command.category, [command as SkyraCommand]);
```

Comment:
Remove the casts as `command` will match the type `Command` once the previous message is addressed.